crates/ty_project/src/metadata/options.rs

```diff
@@ -277,14 +278,36 @@ impl Options {
             roots
         };
 
-        let settings = SearchPathSettings {
-            extra_paths: environment
+        // collect the existing site packages
+        let mut extra_paths: Vec<SystemPathBuf> = Vec::new();
```

Comment:
I think we should only add `PYTHONPATH` if the user didn't explicitly set `extra_paths` (when `extra_paths` is `None`).

Reply:
Hmm yeah I can see that. I was going off of cli-args > env vars but it can be confusing if the names don't match. I think the intention was that PYTHONPATH and --extra-paths both set extra_paths. I can see it either way though I think I do prefer avoiding the implentation leakage and instead just allow both PYTHONPATH and --extra-paths, the `extra` part is pretty telling.